### PR TITLE
python-apt issues

### PR DESCRIPTION
**Pull request by aostromich**
Hello, these are 2 python-apt CVEs from Ubuntu, thank you.

### CODE REVIEW

**Comment by cve-team** [comment]
aostromich is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2019/15xxx/CVE-2019-15795.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "python-apt only checks the MD5 sums of downloaded files in `Version.fetch_binary()` and `Version.fetch_source()` of apt/package.py allowing a man-in-the-middle attack which could potentially be used to install altered packages."
+            }
```
Comment (iamleot) [inline 2019/15xxx/CVE-2019-15795.json]: Please mention affected and fixed versions in the description too.

File: 2019/15xxx/CVE-2019-15796.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        }
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "Python-apt doesn't check if hashes are signed in `Version.fetch_binary()` and `Version.fetch_source()` of apt/package.py or in `_fetch_archives()` of apt/cache.py. This allows downloads from unsigned repositories which shouldn't be allowed."
```
Comment (iamleot) [inline 2019/15xxx/CVE-2019-15796.json]: Please mention affected and fixed versions in the description too.

**Comment by aostromich** [comment]
> Please mention affected and fixed versions in the description too!
> 
> Thanks!
Added, thank you! Please let me know if anything else should be changed.

**Pull request merged**